t/001-tls.t

```diff
@@ -415,8 +415,8 @@ qpvBFOY29Mlww88oWNGTA8QeW84Y1EJbRkHavzSsMFB73sxidQW0cHNC5t9RCKAQ
 uibeZgK1Yk7YQKXdvbZvXwrgTcAjCdbppw2L6e0Uy+OGgNjnIps8K460SdaIiA==
 -----END CERTIFICATE-----
 
---- error_log
-[lua] ssl_certificate_by_lua:2: ssl cert by lua is running!
+--- error_log eval
+qr/ssl_certificate_by_lua.*ssl cert by lua is running!/
```

Comment:
the log was: `[lua] ssl_certificate_by_lua(nginx.conf:30):2: ssl cert by lua is running!`

Reply:
What causes this change? any clue?

Reply:
I am trying to figure it out but not sure yet

Reply:
@chronolaw I think I found it: https://github.com/openresty/lua-nginx-module/pull/2008, see how the tests changed: https://github.com/openresty/lua-nginx-module/pull/2008/files#diff-9f25aa4f9bc139d6b6d65bf14649c6524911dc99ca7fc92e3337867a969bee1dR120